dev/Tiltfile

```diff
@@ -23,13 +23,15 @@ groups = {
         "pinga",
         "rebaser",
         "sdf",
+
         "veritech",
         "rust-initial-build",
     ],
     "frontend": [
         "auth-portal",
         "docs",
         "web",
+        "luminork",
```

Comment:
Interesting that this is in the frontend... what's the reason?

Reply:
It's an API interface to the product :)